index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Harry Potter Memory Game - Starting Page</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"> <!-- Bootstrap -->
+	<link rel="stylesheet" type="text/css" href="css/style.css"> <!-- my style sheet -->
+</head>
+<body>
+
+
+<div class="gradient"> <!-- Main container with backbround image-->
+
+	<audio autoplay loop id="myAudio"> <!-- audio -->
+		<source src="sound/hp.mp3" type="audio/mpeg">
+	</audio> <!-- end of audio -->
+
+	<div class="row">
+		<div class="col-md-6"> <!-- logo -->
+
+			<div class="pl-5 pt-5 mr-0 col-md-6">
+				<img src="img/HarryPotterLogo.png" class="img-logo">
+				<h1 class="h1-logo text-center">memory game</h1>
+			</div>
+		</div> <!-- end of logo -->
+
+		<div class="col-md-6 text-md-right text-sm-center pr-5 pt-5 pl-5 lead icon"> <!-- music icon -->
+			<div>
+				<a href="#" class="speaker"><span></span></a>
+			</div>
+		</div> <!-- end of music icon -->
+	</div> <!-- end of row -->
+	
+
+	<div class="row"> <!-- buttons -->
+		<div class="container col-xs-12 text-center m-auto">
+			<a class="btn btn-warning mt-5 mb-4" href="bl1.html" role="button">Play</a><br>
+			<a class="btn btn-warning mb-4" role="button" data-toggle="modal" data-target="#instructions">Instructions</a><br>
+			<a class="btn btn-warning mb-4" href="#" role="button">Options</a>
```

Comment:
Don't include unfinished features in your final product - this doesn't do anything at the moment, so it shouldn't be here.